bind/motor-mobile/bind.go

```diff
@@ -74,7 +104,7 @@ func Address() string {
 
 // Balance returns the balance of the wallet.
 func Balance() int {
-	return int(instance.Balance())
+	return int(instance.GetBalance())
```

Comment:
should be nil checked for the cast and return. this could panic.

Reply:
`GetBalance` can't return nil